js/index.js

```diff
@@ -4,7 +4,10 @@ const player1 = document.querySelector('.player1-name-input');
 const player2 = document.querySelector('.player2-name-input');
 const player3 = document.querySelector('.player3-name-input');
 const startGameButton = document.querySelector('.start-button');
-var startScreen = document.querySelector('.start-screen');
+const startScreen = document.querySelector('.start-screen');
+const player1Name = document.querySelector('.player1-name');
+const player2Name = document.querySelector('.player2-name');
```

Comment:
It is confusing as to why you have the duplicated consts for players. I can see that lines 4 and 5 as selectin the inputs to get values in line 18, but is unclear what lines 8-10 and why you are selecting another element from a similarly named class.